OK, it looks like that one passed. Now we have

```
Warning, treated as error:
/home/travis/build/sympy/sympy/sympy/codegen/cfunctions.py:docstring of sympy.codegen.cfunctions.exp2.eval:15:Unexpected indentation.
make: *** [html] Error 2
```

Maybe there is a setting to ignore warnings? Or else maybe this one can be fixed like the last one.

--- a/doc/src/modules/codegen.rst
+++ b/doc/src/modules/codegen.rst
@@ -57,7 +57,7 @@ Code printers (sympy.printing)
 
 This is where the meat of code generation is; the translation of SymPy
 expressions to specific languages. Supported languages are C
-(:py:func:`sympy.printing.ccode.ccode`), R 
+(:py:func:`sympy.printing.ccode.ccode`), R
 (:py:func:`sympy.printing.rcode.rcode`), Fortran 95
 (:py:func:`sympy.printing.fcode.fcode`), JavaScript
 (:py:func:`sympy.printing.jscode.jscode`), Julia
@@ -560,17 +560,20 @@ Special (finite precision arithmetic) math functions
 ----------------------------------------------------
 
 .. automodule:: sympy.codegen.cfunctions
-    :members:
+
+   :members:
 
 
 Fortran specific functions
 --------------------------
 
 .. automodule:: sympy.codegen.ffunctions
-    :members:
+
+   :members:
 
 Classes for abstract syntax trees (sympy.codegen.ast)
 -----------------------------------------------------
 
 .. automodule:: sympy.codegen.ast
-    :members:
+
+   :members:

--- a/doc/src/modules/core.rst
+++ b/doc/src/modules/core.rst
@@ -267,6 +267,7 @@ mod
 Mod
 ^^^
 .. autoclass:: Mod
+
    :members:
 
 relational
@@ -276,36 +277,43 @@ relational
 Rel
 ^^^
 .. autoclass:: Rel
+
    :members:
 
 Eq
 ^^
 .. autoclass:: Eq
+
    :members:
 
 Ne
 ^^
 .. autoclass:: Ne
+
    :members:
 
 Lt
 ^^
 .. autoclass:: Lt
+
    :members:
 
 Le
 ^^
 .. autoclass:: Le
+
    :members:
 
 Gt
 ^^
 .. autoclass:: Gt
+
    :members:
 
 Ge
 ^^
 .. autoclass:: Ge
+
    :members:
 
 Equality

--- a/doc/src/modules/functions/combinatorial.rst
+++ b/doc/src/modules/functions/combinatorial.rst
@@ -7,24 +7,28 @@ bell
 ----
 
 .. autoclass:: sympy.functions.combinatorial.numbers.bell
+
    :members:
 
 bernoulli
 ---------
 
 .. autoclass:: sympy.functions.combinatorial.numbers.bernoulli
+
    :members:
 
 binomial
 --------
 
 .. autoclass:: sympy.functions.combinatorial.factorials.binomial
+
    :members:
 
 catalan
 -------
 
 .. autoclass:: sympy.functions.combinatorial.numbers.catalan
+
    :members:
 
 
@@ -32,6 +36,7 @@ euler
 -----
 
 .. autoclass:: sympy.functions.combinatorial.numbers.euler
+
    :members:
 
 
@@ -39,18 +44,21 @@ factorial
 ---------
 
 .. autoclass:: sympy.functions.combinatorial.factorials.factorial
+
    :members:
 
 subfactorial
 ------------
 
 .. autoclass:: sympy.functions.combinatorial.factorials.subfactorial
+
    :members:
 
 factorial2 / double factorial
 -----------------------------
 
 .. autoclass:: sympy.functions.combinatorial.factorials.factorial2
+
    :members:
 
 
@@ -58,18 +66,21 @@ FallingFactorial
 ----------------
 
 .. autoclass:: sympy.functions.combinatorial.factorials.FallingFactorial
+
    :members:
 
 fibonacci
 ---------
 
 .. autoclass:: sympy.functions.combinatorial.numbers.fibonacci
+
    :members:
 
 harmonic
 --------
 
 .. autoclass:: sympy.functions.combinatorial.numbers.harmonic
+
    :members:
 
 
@@ -77,6 +88,7 @@ lucas
 -----
 
 .. autoclass:: sympy.functions.combinatorial.numbers.lucas
+
    :members:
 
 
@@ -84,6 +96,7 @@ MultiFactorial
 --------------
 
 .. autoclass:: sympy.functions.combinatorial.factorials.MultiFactorial
+
    :members:
 
 
@@ -91,6 +104,7 @@ RisingFactorial
 ---------------
 
 .. autoclass:: sympy.functions.combinatorial.factorials.RisingFactorial
+
    :members:
 
 stirling

--- a/doc/src/modules/functions/elementary.rst
+++ b/doc/src/modules/functions/elementary.rst
@@ -12,54 +12,63 @@ re
 --
 
 .. autoclass:: sympy.functions.elementary.complexes.re
+
    :members:
 
 im
 --
 
 .. autoclass:: sympy.functions.elementary.complexes.im
+
    :members:
 
 sign
 ----
 
 .. autoclass:: sympy.functions.elementary.complexes.sign
+
    :members:
 
 Abs
 ---
 
 .. autoclass:: sympy.functions.elementary.complexes.Abs
+
    :members:
 
 arg
 ---
 
 .. autoclass:: sympy.functions.elementary.complexes.arg
+
    :members:
 
 conjugate
 ---------
 
 .. autoclass:: sympy.functions.elementary.complexes.conjugate
+
    :members:
 
 polar_lift
 ----------
 
 .. autoclass:: sympy.functions.elementary.complexes.polar_lift
+
    :members:
 
 periodic_argument
 -----------------
 
 .. autoclass:: sympy.functions.elementary.complexes.periodic_argument
+
    :members:
 
 principal_branch
 -----------------
 
 .. autoclass:: sympy.functions.elementary.complexes.principal_branch
+
    :members:
 
 
@@ -73,42 +82,49 @@ sin
 ---
 
 .. autoclass:: sympy.functions.elementary.trigonometric.sin
+
    :members:
 
 cos
 ---
 
 .. autoclass:: sympy.functions.elementary.trigonometric.cos
+
    :members:
 
 tan
 ---
 
 .. autoclass:: sympy.functions.elementary.trigonometric.tan
+
    :members:
 
 cot
 ---
 
 .. autoclass:: sympy.functions.elementary.trigonometric.cot
+
    :members:
 
 sec
 ---
 
 .. autoclass:: sympy.functions.elementary.trigonometric.sec
+
    :members:
 
 csc
 ---
 
 .. autoclass:: sympy.functions.elementary.trigonometric.csc
+
    :members:
 
 sinc
 ----
 
 .. autoclass:: sympy.functions.elementary.trigonometric.sinc
+
    :members:
 
 
@@ -119,42 +135,49 @@ asin
 ----
 
 .. autoclass:: sympy.functions.elementary.trigonometric.asin
+
    :members:
 
 acos
 ----
 
 .. autoclass:: sympy.functions.elementary.trigonometric.acos
+
    :members:
 
 atan
 ----
 
 .. autoclass:: sympy.functions.elementary.trigonometric.atan
+
    :members:
 
 acot
 ----
 
 .. autoclass:: sympy.functions.elementary.trigonometric.acot
+
    :members:
 
 asec
 ----
 
 .. autoclass:: sympy.functions.elementary.trigonometric.asec
+
    :members:
 
 acsc
 ----
 
 .. autoclass:: sympy.functions.elementary.trigonometric.acsc
+
    :members:
 
 atan2
 -----
 
 .. autoclass:: sympy.functions.elementary.trigonometric.atan2
+
    :members:
 
 
@@ -169,42 +192,49 @@ HyperbolicFunction
 ------------------
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.HyperbolicFunction
+
    :members:
 
 sinh
 ----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.sinh
+
    :members:
 
 cosh
 ----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.cosh
+
    :members:
 
 tanh
 ----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.tanh
+
    :members:
 
 coth
 ----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.coth
+
    :members:
 
 sech
 ----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.sech
+
    :members:
 
 csch
 ----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.csch
+
    :members:
 
 
@@ -215,36 +245,42 @@ asinh
 -----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.asinh
+
    :members:
 
 acosh
 -----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.acosh
+
    :members:
 
 atanh
 -----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.atanh
+
    :members:
 
 acoth
 -----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.acoth
+
    :members:
 
 asech
 -----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.asech
+
    :members:
 
 acsch
 -----
 
 .. autoclass:: sympy.functions.elementary.hyperbolic.acsch
+
    :members:
 
 sympy.functions.elementary.integers
@@ -254,6 +290,7 @@ ceiling
 -------
 
 .. autoclass:: sympy.functions.elementary.integers.ceiling
+
    :members:
 
 
@@ -261,6 +298,7 @@ floor
 -----
 
 .. autoclass:: sympy.functions.elementary.integers.floor
+
    :members:
 
 
@@ -282,18 +320,21 @@ exp
 ---
 
 .. autoclass:: sympy.functions.elementary.exponential.exp
+
    :members:
 
 LambertW
 --------
 
 .. autoclass:: sympy.functions.elementary.exponential.LambertW
+
    :members:
 
 log
 ---
 
 .. autoclass:: sympy.functions.elementary.exponential.log
+
    :members:
 
 
@@ -304,6 +345,7 @@ ExprCondPair
 ------------
 
 .. autoclass:: sympy.functions.elementary.piecewise.ExprCondPair
+
    :members:
 
 
@@ -311,6 +353,7 @@ Piecewise
 ---------
 
 .. autoclass:: sympy.functions.elementary.piecewise.Piecewise
+
    :members:
 
 .. autofunction:: sympy.functions.elementary.piecewise.piecewise_fold
@@ -323,12 +366,14 @@ IdentityFunction
 ----------------
 
 .. autoclass:: sympy.functions.elementary.miscellaneous.IdentityFunction
+
    :members:
 
 Min
 ---
 
 .. autoclass:: sympy.functions.elementary.miscellaneous.Min
+
    :members:
 
 
@@ -336,6 +381,7 @@ Max
 ---
 
 .. autoclass:: sympy.functions.elementary.miscellaneous.Max
+
    :members:
 
 root

--- a/doc/src/modules/functions/special.rst
+++ b/doc/src/modules/functions/special.rst
@@ -4,11 +4,13 @@ Special
 DiracDelta
 ----------
 .. autoclass:: sympy.functions.special.delta_functions.DiracDelta
+
    :members:
 
 Heaviside
 ---------
 .. autoclass:: sympy.functions.special.delta_functions.Heaviside
+
    :members:
 
 Singularity Function
@@ -16,6 +18,7 @@ Singularity Function
 .. module:: sympy.functions.special.singularity_functions
 
 .. autoclass:: sympy.functions.special.singularity_functions.SingularityFunction
+
    :members:
 
 Gamma, Beta and related Functions
@@ -23,19 +26,25 @@ Gamma, Beta and related Functions
 .. module:: sympy.functions.special.gamma_functions
 
 .. autoclass:: sympy.functions.special.gamma_functions.gamma
+
    :members:
 .. autoclass:: sympy.functions.special.gamma_functions.loggamma
+
    :members:
 .. autoclass:: sympy.functions.special.gamma_functions.polygamma
+
    :members:
 .. autofunction:: sympy.functions.special.gamma_functions.digamma
 .. autofunction:: sympy.functions.special.gamma_functions.trigamma
 .. autoclass:: sympy.functions.special.gamma_functions.uppergamma
+
    :members:
 .. autoclass:: sympy.functions.special.gamma_functions.lowergamma
+
    :members:
 .. module:: sympy.functions.special.beta_functions
 .. autoclass:: sympy.functions.special.beta_functions.beta
+
    :members:
 
 Error Functions and Fresnel Integrals
@@ -51,6 +60,7 @@ Error Functions and Fresnel Integrals
 .. autoclass:: sympy.functions.special.error_functions.erf2inv
 
 .. autoclass:: sympy.functions.special.error_functions.FresnelIntegral
+
    :members:
 
 .. autoclass:: fresnels
@@ -75,6 +85,7 @@ Bessel Type Functions
 .. module:: sympy.functions.special.bessel
 
 .. autoclass:: sympy.functions.special.bessel.BesselBase
+
    :members:
 
 .. autoclass:: sympy.functions.special.bessel.besselj
@@ -93,6 +104,7 @@ Airy Functions
 --------------
 
 .. autoclass:: sympy.functions.special.bessel.AiryBase
+
    :members:
 
 .. autoclass:: sympy.functions.special.bessel.airyai
@@ -119,9 +131,11 @@ Riemann Zeta and Related Functions
 Hypergeometric Functions
 ------------------------
 .. autoclass:: sympy.functions.special.hyper.hyper
+
    :members:
 
 .. autoclass:: sympy.functions.special.hyper.meijerg
+
    :members:
 
 Elliptic integrals
@@ -138,6 +152,7 @@ Mathieu Functions
 .. module:: sympy.functions.special.mathieu_functions
 
 .. autoclass:: sympy.functions.special.mathieu_functions.MathieuBase
+
    :members:
 
 .. autoclass:: sympy.functions.special.mathieu_functions.mathieus
@@ -154,6 +169,7 @@ Jacobi Polynomials
 ++++++++++++++++++
 
 .. autoclass:: sympy.functions.special.polynomials.jacobi
+
    :members:
 
 .. autofunction:: sympy.functions.special.polynomials.jacobi_normalized
@@ -162,44 +178,54 @@ Gegenbauer Polynomials
 ++++++++++++++++++++++
 
 .. autoclass:: sympy.functions.special.polynomials.gegenbauer
+
    :members:
 
 Chebyshev Polynomials
 +++++++++++++++++++++
 
 .. autoclass:: sympy.functions.special.polynomials.chebyshevt
+
    :members:
 
 .. autoclass:: sympy.functions.special.polynomials.chebyshevu
+
    :members:
 
 .. autoclass:: sympy.functions.special.polynomials.chebyshevt_root
+
    :members:
 
 .. autoclass:: sympy.functions.special.polynomials.chebyshevu_root
+
    :members:
 
 Legendre Polynomials
 ++++++++++++++++++++
 
 .. autoclass:: sympy.functions.special.polynomials.legendre
+
    :members:
 
 .. autoclass:: sympy.functions.special.polynomials.assoc_legendre
+
    :members:
 
 Hermite Polynomials
 +++++++++++++++++++
 
 .. autoclass:: sympy.functions.special.polynomials.hermite
+
    :members:
 
 Laguerre Polynomials
 ++++++++++++++++++++
 
 .. autoclass:: sympy.functions.special.polynomials.laguerre
+
    :members:
 .. autoclass:: sympy.functions.special.polynomials.assoc_laguerre
+
    :members:
 
 Spherical Harmonics
@@ -219,7 +245,9 @@ Tensor Functions
 .. autofunction:: sympy.functions.special.tensor_functions.eval_levicivita
 
 .. autoclass:: sympy.functions.special.tensor_functions.LeviCivita
+
    :members:
 
 .. autoclass:: sympy.functions.special.tensor_functions.KroneckerDelta
+
    :members:

--- a/doc/src/modules/ntheory.rst
+++ b/doc/src/modules/ntheory.rst
@@ -6,6 +6,7 @@ Number Theory
 Ntheory Class Reference
 -----------------------
 .. autoclass:: Sieve
+
    :members:
 
 Ntheory Functions Reference
@@ -136,10 +137,12 @@ Ntheory Functions Reference
 .. autofunction:: discrete_log
 
 .. automodule:: sympy.ntheory.continued_fraction
-    :members:
+
+   :members:
 
 .. autoclass:: sympy.ntheory.mobius
-    :members:
+
+   :members:
 
 .. module:: sympy.ntheory.egyptian_fraction
 

--- a/doc/src/modules/physics/quantum/state.rst
+++ b/doc/src/modules/physics/quantum/state.rst
@@ -3,4 +3,5 @@ State
 =====
 
 .. automodule:: sympy.physics.quantum.state
+
    :members:

